src/tests/ftest/harness/basic.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python
```

Comment:
Don't we need the license?
```suggestion
#!/usr/bin/python
"""
  (C) Copyright 2018-2021 Intel Corporation.
  SPDX-License-Identifier: BSD-2-Clause-Patent
"""
```

Reply:
Yeah.  Added.  Thanks.